habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -1084,6 +1084,37 @@ def step(self, task: EmbodiedTask, *args: Any, **kwargs: Any):
         return self._sim.get_observations_at()  # type: ignore
 
 
+@registry.register_task_action
+class VelocityStopAction(SimulatorTaskAction):
+    name: str = "velocity_stop"
```

Comment:
Need doc string, why this action is continuous as it's only discrete stop call.

Reply:
@ykarmesh can you comment on this? 
My understanding is that we're expecting the policy to have some sort of a softmax output with scores for each discrete action, and since "stop" is one of the possible actions it will also be determined from a value ranging from -1 to 1.

Reply:
Hi @mathfac. I think the logics is that once the velocity is below the certain threshold, the robot just calls the Stop action. We want to make this consistent across the velocity control